OpenOversight/app/templates/partials/officer_incidents.html

```diff
@@ -0,0 +1,23 @@
+<h3>Incidents</h3>
+{% if officer.incidents %}
+    <ul class="list-group">
+    {% for incident in officer.incidents %}
+        <li class="list-group-item">
+        <h4>
+            <a href="{{ url_for('main.incident_api', obj_id=incident.id)}}">
+                Incident {{ incident.report_number }}
+            </a>
+            {% if current_user.is_administrator or (current_user.is_area_coordinator and current_user.ac_department_id == incident.department_id) %}
+                <a href="{{ url_for('main.incident_api', obj_id=incident.id) + '/edit' }}">
+                    <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
+                </a>
+            {% endif %}
+        </h4>
+        {% include 'partials/incident_fields.html' %}
+        </li>
+    {% endfor %}
+    </ul>
+{% endif %}
+{% if is_admin_or_coordinator %}
+    <a href="{{ url_for('main.incident_api') + 'new?officer_id={}'.format(officer.id) }}" class='btn btn-primary'>New Incident</a>
```

Comment:
I feel like there has to be a better way to add parameters to URLs in templates...
Update: there is! Maybe we can modify this since we're touching it anyways?
https://stackoverflow.com/a/23144200/3277713

Reply:
We can't do that here because we also need to append `new` to the route before adding the query parameters. If we do the way suggested in the SO link we end up appending `new` after the query params, which isn't right.

Reply:
Got it, thanks!